contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,85 @@
+# VueJs
+
+Author: Lu Lechuan
+
+A JavaScript framework is an application framework written in JavaScript. It differs from a JavaScript library in its control flow: a library offers functions to be called by its parent code, whereas a framework defines the entire application design.
+JavaScript frameworks aim to improve code quality and maintainablity, example of some javascript frameworks include Angular, React and Vue.
+
+## Advantages of VueJs
+VueJs is an approachable, versetile and performant framework that helps to create a more maintainable and testable database.
+
+* approachable: VueJs is very easy to learn. Comparing to other framework such as Augular, Vue is simple in term of API and design. Learning enough to build non-trivial applications typically takes less than a day.
+
+* progressive: Vue is designed from the ground up to be incrementally adoptable. The core library is focused on the view layer only, and is easy to pick up and integrate with other libraries or existing projects. This means that if you hava a large application, you can plug Vue into just a part of your application without messing the rest up.
+
+* versetile: Vue is perfectly capable of powering sophisticated Single-Page Applications when used in combination with modern tooling and supporting libraries.
+
+* clean: Vue symtax is simple and this can make the HTML pages very clean.
+
+## Installation
+
+There are many ways to install VueJs.
+* Direct `<script>` Include:
```

Comment:
This header is a bit confusing to me... perhaps a better way to say is "Import vue in the <script> tag of HTML file"?